ts/indicators/wma.src.ts

```diff
@@ -64,21 +62,16 @@ var seriesType = H.seriesType;
  * @private
  */
 function accumulateAverage(
-    points: Array<Array<(number|null|undefined)>>,
+    points: Array<Array<number>>,
     xVal: Array<number>,
-    yVal: Array<(
-        number|
-        Array<(number|null|undefined)>|
-        null|
-        undefined
-    )>,
+    yVal: Array<Array<number>>,
     i: number,
     index: number
 ): void {
-    var xValue = xVal[i],
-        yValue = index < 0 ? yVal[i] : (yVal[i] as any)[index];
+    var xValue: number = xVal[i],
+        yValue: (number|Array<number>) = index < 0 ? yVal[i] : yVal[i][index];
 
-    points.push([xValue, yValue]);
+    points.push([xValue, (yValue as any)]);
```

Comment:
I have the impression that this is not correct. Actually points can be `Array<[number, (number|Array<number>)]>`, if the yValues can be either `number` or `Array<number>`

Reply:
Thanks, that was nicely spotted. I totally missed that one.